src/transformers/pipelines/base.py

```diff
@@ -1164,7 +1164,7 @@ def __call__(self, inputs, *args, num_workers=None, batch_size=None, **kwargs):
 
         self.call_count += 1
         if self.call_count > 10 and self.framework == "pt" and self.device.type == "cuda":
-            warnings.warn(
+            logger.warning(
```

Comment:
I'd recommend using `warning_once` here. This way, the warning will only display the first time it's hit, so shouldn't spam the logs. This will hopefully discourage having to change the logging level to get it to disappear (who knows what other useful warnings you might be surpressing! :D )

```suggestion
            logger.warning_once(
```